src/ox_modules/module-web.js

```diff
@@ -286,7 +289,8 @@ export default class WebModule extends WebDriverModule {
                         const body = "{\"passed\":"+passed+"}";
 
                         const myAccount = new SauceLabs({ user: username, key: accessKey});
-                        myAccount.updateJob(username, id, body);
+                        await myAccount.updateJob(username, id, body);
```

Comment:
This doesn't seem to be related to the current commit?

Reply:
Fixed